src/main/webapp/js/instructorFeedbackResultsAjaxByQuestion.js

```diff
@@ -50,11 +51,23 @@ $(document).ready(function() {
                 $(panelHeading).click(toggleSingleCollapse);
                 $(panelHeading).trigger('click');
 
+                if (isPanelSetAsEmptyByBackend($panelBody)) {
+                    displayAsEmptyPanel($panelBody);
+                }
+
                 showHideStats();
             }
         });
     };
 
+    var isPanelSetAsEmptyByBackend = function($panelBody) {
+        return $panelBody.find('.no-response').length !== 0;
+    };
+
+    var displayAsEmptyPanel = function($panelBody) {
+        $panelBody.parent().parent().removeClass('panel-info').addClass('panel-default');
```

Comment:
This .parent chain will break easily if the structure of the html changes. Find another way to reference the panel in a clearer way?